res/skins/Shade/style.qss

```diff
@@ -344,7 +344,8 @@ WCueMenuPopup QLabel {
   selection-background-color: #ccc;
   padding: 2px;
 }
-#CueColorPicker QPushButton {
+#CueColorPicker QPushButton,
+WLibrary WColorPicker QPushButton {
   width: 38px;
```

Comment:
How about making the track color picker buttons square (20x20px), because track color palettes are probably bigger than hotcue color palettes.

Reply:
IIRC Shade is the only skin where I set the dimensions of the color buttons explicitely. In the others they're sized according to OS theme or Qt rules, which is a wide rectangle from what I've seen.
Do the buttons get resized when the palettes are expanded by users to beyond 8/9 buttons? I doubt so because with 8+1 colors it's the track color picker popup that expands.
Also I don't assume users who modify palettes necessarily stick to 8 hotcue colors just because Mixxx skin support up to 8 hotcues by default; both hotue and track color palettes can grow.
Thus I suggest we stick to the wide rectangular shape.

Reply:
Do we even have to set the dimensions explicitly? The columns are not fixed, it can be 3-5 columns depending on the number of items. This because the last row looks ugly when it only contains a single element, so determiney the best value so that the last row has the least missing items.

Reply:
Ok, I'll just merge know because this is not really related to this branch.

Reply:
Jup. I'm still trying to build with cmake so I can test branches quicker.
We can check this later on.